core/src/main/java/org/infinispan/distexec/DistributedExecutorService.java

```diff
@@ -45,6 +46,7 @@
  * @author Vladimir Blagojevic
  *
  * @since 5.0
+ * @deprecated since 9.1 Please use {@link EmbeddedCacheManager#executor()} or {@link org.infinispan.CacheStream} instead.
```

Comment:
For symmetry, I'd either replace `{@link EmbeddedCacheManager#executor()}` with `{@link ClusterExecutor}`, or `{@link org.infinispan.CacheStream}` with `{@link org.infinispan.CacheCollection#stream()}`.